pom.xml

```diff
@@ -86,6 +86,12 @@
 			<artifactId>jackson-databind</artifactId>
 			<version>2.9.10.5</version>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.maven.plugins</groupId>
```

Comment:
Why this ?

Reply:
used this liabrary for fileUtils to read contacts from file